asterisk_click2dial/models/asterisk_server.py

```diff
@@ -146,29 +146,33 @@ def _get_connect_info(self, url_path):
         user = self.env.user
         ast_server = user.get_asterisk_server_from_user()
         auth = (ast_server.login, ast_server.password)
-        url = "http://%s:%s%s" % (ast_server.ip_address, ast_server.port, url_path)
+        url = f"http://{ast_server.ip_address}:{ast_server.port}{url_path}"
         return ast_server, auth, url
 
     def test_ari_connection(self):
         self.ensure_one()
         auth = (self.login, self.password)
-        url = "http://%s:%s/ari/asterisk/info" % (self.ip_address, self.port)
+        url = f"http://{self.ip_address}:{self.port}/ari/asterisk/info"
         try:
             res = requests.get(url, auth=auth, timeout=TIMEOUT)
         except Exception as e:
             raise UserError from None(
-                _("Connection Test Failed! The error message is: %s" % e)
+                _(f"Connection Test Failed! The error message is: {e}")
             )
         if res.status_code != 200:
             raise UserError(
                 _("Connection Test Failed! HTTP error code: %s" % res.status_code)
             )
-        raise UserError(
-            _(
-                "Connection Test Successfull! Odoo can successfully login to "
-                "the Asterisk Manager Interface."
-            )
-        )
+
+        return {
+            "type": "ir.actions.client",
+            "tag": "display_notification",
+            "params": {
+                "type": "success",
+                "message": _("Your request has been successfully sent"),
```

Comment:
Why did you change the message ?
IMHO, the original one is better because the connexion is towards AMI